[2013-02-26 10:05:21,047][INFO ][node                     ] [Mammomax] {0.20.1}[33014]: stopping ...
[2013-02-26 10:05:21,163][INFO ][node                     ] [Mammomax] {0.20.1}[33014]: stopped
[2013-02-26 10:05:21,163][INFO ][node                     ] [Mammomax] {0.20.1}[33014]: closing ...
[2013-02-26 10:05:21,173][INFO ][node                     ] [Mammomax] {0.20.1}[33014]: closed
[2013-02-26 10:11:18,119][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: initializing ...
[2013-02-26 10:11:18,155][INFO ][plugins                  ] [Schmidt, Johann] loaded [], sites []
[2013-02-26 10:11:37,555][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: initialized
[2013-02-26 10:11:37,556][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: starting ...
[2013-02-26 10:11:40,071][INFO ][transport                ] [Schmidt, Johann] bound_address {inet[/0:0:0:0:0:0:0:0%0:9300]}, publish_address {inet[/172.16.201.235:9300]}
[2013-02-26 10:11:43,281][INFO ][cluster.service          ] [Schmidt, Johann] new_master [Schmidt, Johann][mI_lwhGNR6qmQ9SLxy5wEQ][inet[/172.16.201.235:9300]], reason: zen-disco-join (elected_as_master)
[2013-02-26 10:11:43,737][INFO ][discovery                ] [Schmidt, Johann] elasticsearch/mI_lwhGNR6qmQ9SLxy5wEQ
[2013-02-26 10:11:43,817][INFO ][http                     ] [Schmidt, Johann] bound_address {inet[/0:0:0:0:0:0:0:0%0:9200]}, publish_address {inet[/172.16.201.235:9200]}
[2013-02-26 10:11:43,818][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: started
[2013-02-26 10:11:47,269][INFO ][gateway                  ] [Schmidt, Johann] recovered [1] indices into cluster_state
[2013-02-26 11:37:23,923][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: stopping ...
[2013-02-26 11:37:24,011][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: stopped
[2013-02-26 11:37:24,011][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: closing ...
[2013-02-26 11:37:24,027][INFO ][node                     ] [Schmidt, Johann] {0.20.1}[473]: closed
[2013-02-26 15:05:12,579][INFO ][node                     ] [Golem] {0.20.1}[1623]: initializing ...
[2013-02-26 15:05:12,587][INFO ][plugins                  ] [Golem] loaded [], sites []
[2013-02-26 15:05:15,353][INFO ][node                     ] [Golem] {0.20.1}[1623]: initialized
[2013-02-26 15:05:15,356][INFO ][node                     ] [Golem] {0.20.1}[1623]: starting ...
[2013-02-26 15:05:15,507][INFO ][transport                ] [Golem] bound_address {inet[/0:0:0:0:0:0:0:0%0:9300]}, publish_address {inet[/172.16.200.239:9300]}
[2013-02-26 15:05:18,539][INFO ][cluster.service          ] [Golem] new_master [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]], reason: zen-disco-join (elected_as_master)
[2013-02-26 15:05:18,576][INFO ][discovery                ] [Golem] elasticsearch/EMEdo6eiQMqP9X3vhE76RQ
[2013-02-26 15:05:18,596][INFO ][http                     ] [Golem] bound_address {inet[/0:0:0:0:0:0:0:0%0:9200]}, publish_address {inet[/172.16.200.239:9200]}
[2013-02-26 15:05:18,597][INFO ][node                     ] [Golem] {0.20.1}[1623]: started
[2013-02-26 15:05:19,308][INFO ][gateway                  ] [Golem] recovered [1] indices into cluster_state
[2013-02-26 18:53:59,733][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x75df41ed, /172.16.200.239:54039 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,751][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x74f2ef0d, /172.16.200.239:54034 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,751][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xbbee5719, /172.16.200.239:54041 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,736][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xfde0025e, /172.16.200.239:54036 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,765][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x9018befc, /172.16.200.239:54040 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,735][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xeef16bfa, /172.16.200.239:54034 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,770][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xc7c1921f, /172.16.200.239:54038 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,735][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xd2f3e024, /172.16.200.239:54041 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,734][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xf0cc01d1, /172.16.200.239:54035 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,776][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x18c7285a, /172.16.200.239:54039 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,734][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xff2dd73f, /172.16.200.239:54036 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,764][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x224d2bef, /172.16.200.239:54037 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,783][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x1f89f1aa, /172.16.200.239:54033 => /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:53:59,760][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x8c59e0ac, /172.16.200.239:54038 :> /172.16.200.239:9300]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:59)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.processSelectedKeys(AbstractNioWorker.java:471)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:332)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 18:57:59,964][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:02:00,329][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:06:00,621][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:10:00,917][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:14:01,226][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:18:01,527][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:22:01,865][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:26:02,168][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:30:02,467][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:34:02,766][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:38:03,066][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:42:03,370][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:46:03,666][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:50:03,974][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:54:04,327][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 19:58:04,626][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:02:04,947][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:06:05,255][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:10:05,567][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:14:05,867][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:18:06,166][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:22:06,466][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:26:07,324][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:30:07,623][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:34:07,968][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:38:08,323][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:42:08,619][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:46:08,916][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:50:09,269][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:54:09,565][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 20:58:09,869][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:02:10,165][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:05:41,164][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x320bc596]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:652)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,167][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xef83e3eb]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:652)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,168][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xdb3ba141]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,169][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xa0ebc816]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,170][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x9f72e094]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,172][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x250f1528]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,174][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x631b2d1c]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,175][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xbc61559c]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:05:41,176][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x31e37f3b]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:658)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:21,796][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x2f0d42ff]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:652)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,121][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x647df1a4]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:652)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,132][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x735a302f]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,134][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x295a6822]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,136][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xb4d6ffb0]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,139][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x16b4b952]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,141][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x0fe46095]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,144][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x6160f731]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:655)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,146][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xdff523db]], closing connection
java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:658)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-26 21:19:22,147][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.NoRouteToHostException: No route to host
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:148)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:111)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:568)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:642)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:652)
	... 7 more
[2013-02-26 21:23:22,468][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:27:22,765][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:31:23,067][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:35:23,366][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:39:23,667][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:43:23,966][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:47:24,268][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:51:24,568][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:55:24,923][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 21:59:25,220][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:03:25,526][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:07:25,820][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:11:26,120][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:15:26,420][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:19:26,720][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:23:27,020][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:27:27,322][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:31:27,669][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:35:28,126][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:39:28,429][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:43:28,768][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:47:29,065][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:51:29,364][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:55:29,669][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 22:59:29,968][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:03:30,267][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:07:30,566][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:11:30,865][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:15:31,669][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:19:31,967][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:23:32,268][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:27:32,566][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:31:32,864][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:35:33,165][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:39:33,469][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:43:33,770][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:47:34,067][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:51:34,366][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:55:34,668][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-26 23:59:34,967][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
